docs/data/product/dea-surface-reflectance-sentinel-2a-msi/_tables.yaml

```diff
@@ -51,7 +51,7 @@ bands_table:
     nodata: -999
     units: null
     type: int16
-    description: "NBART surface reflectance of the near-infrared range used for detecting vegetation. Equivalent wavelength: 704.1 nm; Bandwidth: 14 nm."
+    description: "NBART surface reflectance of the near-infrared (NIR) range used for detecting vegetation. Equivalent wavelength: 704.1 nm; Bandwidth: 14 nm."
```

Comment:
These bands can be used for anything, not just detecting vegetation. Perhaps "**useful** for detecting vegetation" might be better?